x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_to_dashboard_drilldown/components/dashboard_drilldown_config/i18n.ts

```diff
@@ -12,3 +12,17 @@ export const txtChooseDestinationDashboard = i18n.translate(
     defaultMessage: 'Choose destination dashboard',
   }
 );
+
+export const txtUseCurrentFilters = i18n.translate(
+  'xpack.dashboard.components.DashboardDrilldownConfig.useCurrentFilters',
+  {
+    defaultMessage: 'Use filters and query from origin dashboard',
```

Comment:
Original wording was: `Use filters from current dashboard`, 

We discussed this on one of dev meeting Alona. 
Concern we had with dev team:
1. By just `filters` we understood filter pills, but not search query in the query bar. But we also agreed to pass along query string. so we hoped `filters and query` would be better as it is more explicit
2. We had push back on word `current` because in context of editing drilldown it wasn’t clear what we mean by current. Is it dashboard which we are in or is it currently selected dashboard in Combobox? So we decided to try to use `origin` instead. Like pair: `origin <-> destination`

Hope this is OK

Reply:
Thanks!